### ISSUE TITLE
Unable to install via npm due to reliance on long deprecated zlib which is dependent on node-waf

### ISSUE THREAD

**Issue by gitsharki** [SebastianS90/webpack-polyfill-injector#10]
Can't install with npm, and don't see a workaround:

```
ghost@vdeb:~/dev/precursor$ npm install webpack-polyfill-injector --save-dev
npm WARN deprecated babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read babeljs.io/env to update!
npm WARN deprecated coffee-script@1.9.0: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)
npm WARN deprecated coffee-script@1.12.7: CoffeeScript on NPM has moved to "coffeescript" (no hyphen)

> zlib@1.0.5 preinstall /home/ghost/dev/precursor/node_modules/.staging/zlib-e6ec4d4d
> node-waf clean || true; node-waf configure build

sh: 1: node-waf: not found
sh: 1: node-waf: not found
/home/ghost/dev/precursor
└── (empty)

npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.1.2 (node_modules/watchpack/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.1.2 (node_modules/webpack-dev-server/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.3: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN ajv-keywords@3.1.0 requires a peer of ajv@^6.0.0 but none was installed.
npm ERR! Linux 4.9.0-6-amd64
npm ERR! argv "/home/ghost/local/bin/node" "/home/sharki/local/bin/npm" "install" "webpack-polyfill-injector" "--save-dev"
npm ERR! node v6.11.4
npm ERR! npm  v3.10.10
npm ERR! file sh
npm ERR! code ELIFECYCLE
npm ERR! errno ENOENT
npm ERR! syscall spawn

npm ERR! zlib@1.0.5 preinstall: `node-waf clean || true; node-waf configure build`
npm ERR! spawn ENOENT
npm ERR!
npm ERR! Failed at the zlib@1.0.5 preinstall script 'node-waf clean || true; node-waf configure build'.
npm ERR! Make sure you have the latest version of node.js and npm installed.
npm ERR! If you do, this is most likely a problem with the zlib package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     node-waf clean || true; node-waf configure build
npm ERR! You can get information on how to open an issue for this project with:
npm ERR!     npm bugs zlib
npm ERR! Or if that isn't available, you can get their info via:
npm ERR!     npm owner ls zlib
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     /home/ghost/dev/precursor/npm-debug.log
npm ERR! code 1
```

**Comment by SebastianS90** [comment]
Sorry for the delay.
This will probably be fixed together with #12, as `polyfill-library` does not depend on `zlib`.

Can you please try `npm install webpack-polyfill-injector@next`? This should give you `webpack-polyfill-injector` version `1.0.2-beta.0` which uses the new `polyfill-library` package.

**Comment by SebastianS90** [comment]
Should be fixed in [version 1.0.2](https://github.com/SebastianS90/webpack-polyfill-injector/releases/tag/v1.0.2). Otherwise please let me know...
